### ISSUE TITLE
Emails sent unencrypted by mistake

### ISSUE THREAD

**Issue by AlexAltea** [dejavusecurity/OutlookPrivacyPlugin#165]
Using latest release as of today, i.e. Beta 56, and checking _Encrypt_ flag in the plugin's pane. I got asked for my private key password and after typing it the email got sent... **unencrypted**. 

If something failed during the process, why wasn't a corresponding alert raised before placing the email in the outbox? If only I had knew this earlier I would have aborted the process immediately.

Unfortunately, I'm not able to replicate the process.

**Comment by Meddington** [comment]
So this happened once but you cannot reproduce the issue?

**Comment by AlexAltea** [comment]
@Meddington Exactly. It's really frustrating, I've tried repeating the process again and again with no luck.

I can confirm that I **did** mark both _Sign_ and _Encrypt_ options, and that I got prompted to enter my password for that matter. In fact Outlook shows `-----BEGIN PGP MESSAGE-----` as a preview of that message, but the actual contents are in plaintext (without the BEGIN/END tags). When I see the raw email sent from the Gmail client I also see no references to BEGIN/END tags, everything was sent unencrypted.

![](http://phi.nz/pgp-fail.png)

I really need to find what caused this to happen, as I can't simply jeopardize my privacy.
